ide/src/test/java/com/devonfw/tools/ide/integrationtest/UrlUpdaterMockSingle.java

```diff
@@ -0,0 +1,31 @@
+package com.devonfw.tools.ide.integrationtest;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import com.devonfw.tools.ide.url.model.folder.UrlRepository;
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.AbstractUrlUpdater;
+import com.devonfw.tools.ide.url.updater.JsonUrlUpdater;
+import com.devonfw.tools.ide.url.updater.UrlUpdater;
+
+/**
+ * {@link JsonUrlUpdater} test mock for {@link UrlUpdater} using a single urlVersion.
```

Comment:
same here. Seems like some developer did a lot of Copy&Paste of this comment.
This is not a JsonUrlUpdater.

Reply:
Adjusted javadocs.